help.html

```diff
@@ -26,7 +26,7 @@
     
      <script>
         $(document).ready(function() {
-            var paramsToCheck = ["snippets","blocks","share"];
+            var paramsToCheck = ["snippets","blocks","share","options"];
```

Comment:
options is now part of the beta version, and so it comes with the "experimental" flag.

Reply:
Autocompletion is also part of the experimental flag.

Reply:
done in https://github.com/bbcmicrobit/PythonEditor/pull/199/commits/715543570e673cd8361fce4aff56f62d3b59bc66